src/gui/Lv2ViewBase.cpp

```diff
@@ -51,13 +52,51 @@ namespace lmms::gui
 {
 
 
-Lv2ViewProc::Lv2ViewProc(QWidget* parent, Lv2Proc* ctrlBase, int colNum) :
-	LinkedModelGroupView (parent, ctrlBase, colNum)
+static int numDigitsAsInt(float f)
```

Comment:
I think this function can be in a different place.

Reply:
What about LcdSpinBox.cpp? Or do you see other applications outside of LcdSpinBox, which justifies putting it into the core?

Reply:
> do you see other applications outside of LcdSpinBox

No for now, but I can't find a reason we shouldn't, either.

Reply:
It will be put in `lmms_math.h`, as people on discord suggested.